source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Provider Physical Network
+category: feature
+authors: phoracek,amusil
+feature_name: Provider Physical Network
+feature_modules: engine,vdsm,ovn-provider
+feature_status: In Development
+---
+
+# Provider Physical Network
+
+## Summary
+
+This feature extends [ovirt-ovn-provider](https://www.ovirt.org/develop/release-management/features/network/ovirt-ovn-provider/) supporting physical network access. With this feature, user will be able connect external OVN network to a physical (oVirt native) network.
+
+Progress of the feature is tracked on [ovn-localnet Trello](https://trello.com/b/lxIBEn0A/ovn-localnet). Related patches could be found on [gerrit topic:localnet](https://gerrit.ovirt.org/#/q/topic:localnet).
+
+### Owner
+
+- Feature Owner: Petr Horáček (phoracek)
+- E-mail: <phoracek@redhat.com>
+
+### Benefit to oVirt
+
+With this feature existing OVS VM networking could be replaced with OVN, providing traditional oVirt networking. That would also allow us to drop a part of VM live migration hook.
```

Comment:
With this feature existing OVS VM networking could be replaced with OVN.
->
Existing OVS VM networking could be replaced with OVN.

providing traditional oVirt networking.
-> replacing the legacy, bridge based oVirt networking.
(am I getting the intended meaning right?)

Reply:
I'm not sure, i'm trying to say that current OVS switch type networking will use OVN for VM networks rather than OVS bridging.